From 5c388630198bfd0e1396b4823fdb828a61b6c985 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 27 Jun 2024 11:19:32 -0700
Subject: [PATCH] Docs/mm/index: rename 'Legacy Documentation' to 'Unsorted
 Documentation'

The intention of 'Legacy Documentation' chapter is to keep the old
documents that not yet sorted into the new outline, and encourage new
documents to be integrated in the new outline from the beginning.
However, the new outline will take more time to be completed, and new
documents would take hard time to decide where it should be added since
there is no clear guideline for every case.  For example, memory
allocation profiling document has added to the bottom of the new
outline.  Apparently it was not following some nice guideline or a
result of a discussion.  Also, the title ("legacy") makes people feel
the documents on the chapter might be outdated or not actively
maintained.

Rename 'Legacy Documentation' to 'Unsorted Documentation' and remove the
description saying it is for 'older' documents.  With this change, new
documents that not clear enough where it should be added on the new
outline can be added on the chapter.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/index.rst | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/Documentation/mm/index.rst b/Documentation/mm/index.rst
index 5007815405f1..15614df2b989 100644
--- a/Documentation/mm/index.rst
+++ b/Documentation/mm/index.rst
@@ -25,14 +25,14 @@ see the :doc:`admin guide <../admin-guide/mm/index>`.
    oom
    allocation-profiling
 
-Legacy Documentation
-====================
+Unsorted Documentation
+======================
 
-This is a collection of older documents about the Linux memory management
-(MM) subsystem internals with different level of details ranging from
-notes and mailing list responses for elaborating descriptions of data
-structures and algorithms.  It should all be integrated nicely into the
-above structured documentation, or deleted if it has served its purpose.
+This is a collection of unsorted documents about the Linux memory management
+(MM) subsystem internals with different level of details ranging from notes and
+mailing list responses for elaborating descriptions of data structures and
+algorithms.  It should all be integrated nicely into the above structured
+documentation, or deleted if it has served its purpose.
 
 .. toctree::
    :maxdepth: 1
-- 
2.39.2

